go/oasis-node/cmd/node/node.go

```diff
@@ -111,9 +110,6 @@ type Node struct {
 	Identity   *identity.Identity
 	Beacon     beacon.Backend
 	Epochtime  epochtime.Backend
-	Registry   registryAPI.Backend
```

Comment:
If you're at it, why only some of the backends? What about beacon, epochtime, staking and key manager?